diff -Naur elfutils-0.166/libdwfl/dwfl_line_comp_dir.c elfutils-0.166/libdwfl/dwfl_line_comp_dir.c
--- elfutils-0.166/libdwfl/dwfl_line_comp_dir.c	2017-04-15 20:04:35.297709497 -0400
+++ elfutils-0.166/libdwfl/dwfl_line_comp_dir.c	2017-04-15 20:00:16.094721505 -0400
@@ -26,6 +26,8 @@
    the GNU Lesser General Public License along with this program.  If
    not, see <http://www.gnu.org/licenses/>.  */
 
+#include <stddef.h>
+#include <sys/stat.h>
 #include "libdwflP.h"
 #include <dwarf.h>
 
diff -Naur elfutils-0.166/libdwfl/linux-kernel-modules.c elfutils-0.166/libdwfl/linux-kernel-modules.c
--- elfutils-0.166/libdwfl/linux-kernel-modules.c	2017-04-15 20:04:35.297709497 -0400
+++ elfutils-0.166/libdwfl/linux-kernel-modules.c	2017-04-15 20:00:01.042722203 -0400
@@ -29,6 +29,8 @@
 /* We include this before config.h because it can't handle _FILE_OFFSET_BITS.
    Everything we need here is fine if its declarations just come first.  */
 
+#include <stddef.h>
+#include <sys/stat.h>
 #include <fts.h>
 
 #include <config.h>
